### PR TITLE
minor fixup and adding instructions for working with the updated ikfa…

### PR DESCRIPTION
**Pull request by mlautman**
This PR adds documentation for working with the updated IKFast templates. https://github.com/ros-planning/moveit/pull/1014 

These improvements are the result of a collaboration between [Carbon Robotics](http://www.carbon.ai/) and [PickNik Robotics](picknik.ai)

### CODE REVIEW

File: doc/ikfast/ikfast_tutorial.rst [hunk 1]
```diff
@@ -58,17 +59,19 @@ You should *not* have mpmath installed: ::
 
 MoveIt! IKFast Installation
 ---------------------------
-Install the MoveIt! IKFast package either from debs or from source.
+Install the MoveIt! IKFast package either from Debian or from source.
```
Comment (rhaschke) [inline doc/ikfast/ikfast_tutorial.rst]: Keep this as is. .deb is the file format of Debian packages, which are also used by Ubuntu and probably other distros.

File: doc/ikfast/ikfast_tutorial.rst [hunk 2]
```diff
@@ -216,31 +228,35 @@ For a 7 dof arm, you will need to specify a free link: ::
 The speed and success of this process will depend on the complexity of your robot. A typical 6 DOF manipulator with 3 intersecting axis at the base or wrist will take only a few minutes to generate the IK.
 
 **Known issue**
---freeindex argument is known to have a bug that it cannot handle tree index correctly.
-Say --baselink=2 --eelink=16 and links index from 3 to 9 is not related to current planning group chain. In that case --freeindex will expect index 2 as link 2, but index 3 as link 10 ... and index 9 as link 16.
+There is a known bug that causes the ``--freeindex`` argument to incorrectly handle tree indexes correctly. Say ``--baselink=2``, ``--eelink=16`` and the links indexed from 3 to 9 are not related to the planning group chain. In that case ``--freeindex`` will correctly interpret index 2 as link 2, but incorrectly interpret index 3 as link 10, index 4 as link 11, ... and index 9 as link 16.
```
Comment (rhaschke) [inline doc/ikfast/ikfast_tutorial.rst]: ... incorrectly handle ... correctly?

**Comment by davetcoleman** [comment]
ping @mlautman

**Comment by davetcoleman** [comment]
This feature was merged in 22 days ago so we should address the tutorial now: https://github.com/ros-planning/moveit/pull/1320

File: doc/ikfast/ikfast_tutorial.rst (hunk 2 above)
Comment (mlautman) [reply doc/ikfast/ikfast_tutorial.rst]: good catch

File: doc/ikfast/ikfast_tutorial.rst (hunk 1 above)
Comment (mlautman) [reply doc/ikfast/ikfast_tutorial.rst]: debs is an abbreviation. Debian packages is more clear. It is also the preferred language in the ROS tutorials. http://wiki.ros.org/melodic/Installation/Ubuntu

**Comment by mlautman** [comment]
target changed to master

**Pull request merged**